README.md

```diff
@@ -170,6 +170,31 @@ reporting. [The description of the API][airbrake-api] is available online.
 Additionally, the Rails integration offers automatic exception reporting in any
 Rake tasks<sup>[[link](#rake)]</sup> and [Rails runner][rails-runner].
 
+#### Integration with filter_parameters
+
+If you want to reuse `Rails.application.config.filter_parameters` in Airbrake
+you can configure your notifier the following way:
+
+```rb
+# config/initializers/airbrake.rb
+Airbrake.configure do |c|
+  c.blacklist_keys = Rails.application.config.filter_parameters
+end
+```
+
+There are a few important details:
+
+1. You must load `filter_parameter_logging.rb` before the Airbrake config
+2. If you use Procs to configure `filter_parameters`, you need to convert them
+   to Lambdas. Otherwise you will get `ArgumentError`
+3. If you use Procs to configure `filter_parameters`, the procs must return an
```

Comment:
Nitpick: should it be `Procs` -> `Lambdas` since you mentioned in (2) they should convert to Lambdas?